modulefiles/modulefile.ProdGSI.hera

```diff
@@ -17,29 +23,30 @@ setenv CRTM_FIX /scratch2/NCEPDEV/nwprod/NCEPLIBS/fix/crtm_v2.3.0
 module use /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/modulefiles/stack
 module load hpc/1.1.0
 
-# Load compiler, mpi, cmake, and hdf5/netcdf
+# Load intel compiler and mpi
 module load hpc-intel/18.0.5.274
-
-# python
-module use -a /contrib/anaconda/modulefiles
-module load anaconda/2.3.0 
-
 module load hpc-impi/2018.0.4
 
-module use -a /contrib/cmake/modulefiles
-module load cmake/3.9.0
+# Load production utilities
+module load prod_util/1.2.2
 
-# Load libraries
-module load bacio/2.4.1
+# Load nceplibs
 module load bufr/11.4.0
-module load crtm/2.3.0
 module load ip/3.3.3
 module load nemsio/2.5.2
-module load prod_util/1.2.2
 module load sfcio/1.4.1
 module load sigio/2.3.2
 module load sp/2.3.3
-module load w3emc/2.7.3
 module load w3nco/2.4.1
-module load hdf5/1.10.6
+module load w3emc/2.7.3
+module load bacio/2.4.1
+module load crtm/2.3.0
 module load netcdf/4.7.4
+
+# Load cmake
+module use -a /contrib/cmake/modulefiles
+module load cmake/3.16.1
+
+# Load python
+module use -a /contrib/anaconda/modulefiles
+module load anaconda/2.3.0
```

Comment:
Its best to use and load any system modules before hpc-stack.

Reply:
All suggested changes have been adopted and committed forked master with Mike's help.